test/cypress/integration/group-member-removal.spec.js

```diff
@@ -180,9 +180,11 @@ describe('Group - Removing a member', () => {
     getProposalItems().eq(0).within(() => {
       cy.getByDT('typeDescription').should('contain', `Remove userbot-${userId} from the group.`)
       cy.getByDT('voteFor').click()
-      cy.getByDT('statusDescription')
-        .should('contain', 'Proposal accepted')
     })
+    // see explanatory comment in group-proposals.spec.js for this .pipe() thing
+    cy.get('body')
+      .pipe($el => $el.find('[data-test="proposalsWidget"]').children().eq(0).find('[data-test="statusDescription"]'))
```

Comment:
Why do we have to start with `cy.get('body').pipe($body => ...)` instead of `cy.getByDT('[data-test="proposalsWidget"]').pipe($widget => ...)`?

Reply:
I assume you meant `cy.getByDT('proposalsWidget').pipe(...)`, but yeah, I think it might be possible to start with that. The main thing is that `.children()` be inside of the `pipe`, since that's what can change.

Reply:
So it t urns out this is necessary for some of the integration tests. Upon trying with `cy.getByDT('proposalsWidget')` the `group-chat.spec.js` test began to fail repeatedly.

I think the issue is that sometimes `proposalsWidget` doesn't exist in the DOM (when there are no proposals). So we need that part to be inside of the `.pipe()`.